src/openfermion/utils/_testing_utils.py

```diff
@@ -23,6 +23,16 @@
                              QuadraticHamiltonian)
 
 
+def haar_random_vector(n, seed=None):
+    """Generate an n dimensional Haar randomd vector."""
+    if seed is not None:
+        numpy.random.seed(seed)
+    vector = numpy.random.randn(n).astype(numpy.complex64)
```

Comment:
Why are you using single-precision? We don't do that anywhere else in the library.